Source/Fuse.Common/Tests/FuseTest/TestBase.uno

```diff
@@ -105,6 +105,57 @@ namespace FuseTest
 			}
 		}
 
+		static public T[] GetChildren<T>( Visual v ) where T : class
+		{
+			var list = new List<T>();
+			for (int i=0; i < v.Children.Count; ++i)
+			{
+				var q = v.Children[i] as T;
+				if (q != null)
+					list.Add(q);
+			}
+			return list.ToArray();
+		}
+		
+		static string ConcatList(string a, string b)
```

Comment:
OK, I guess that's fair enough... I'd kind of expect the input to be a list as well (or even varargs), but whatever, this'll do for now ;)